booklist.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Book List</title>
+    <link rel="stylesheet" href="bookliststyle.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src=""></script>
+</head>
+
+<body>

```

Comment:
use the layout of the index.html instead of rewriting the whole styles and include the header and footer also.

Reply:
Okay I am working on it.

Reply:
still it has its own layout instead of reusing the styles.